app/MMB/src/components/MenuGroup.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Color, StyleSheet, Text } from '@kiwicom/mobile-shared';
+import { Translation } from '@kiwicom/mobile-localization';
+
+import MenuItem from './MenuItem';
+
+type Props = {|
+  title: React.Element<typeof Translation>,
+  children: $ReadOnlyArray<React.Element<typeof MenuItem>>,
+|};
+
+/**
+ * This component wraps <MenuItem/> and adds group title and visual separators.
+ * Example:
+ *
+ *
+ *   Trip Info
+ * .------------------------------.
+ * |                              |
+ * |  ICON                        |
+ * |                              |
+ * |        ----------------------|
+ * |                              |
+ * |  ICON                        |
+ * |                              |
+ * `------------------------------`
+ */
+export default function MenuGroup(props: Props) {
+  let separator = null;
```

Comment:
Use const when not reassigning

Which reminds me, we should add
https://eslint.org/docs/rules/prefer-const